src/lib/cfa/ntp_conf.rb

```diff
@@ -130,6 +142,7 @@ def delete(record)
         reset_cache
       end
 
+      # Removes all records that satisfy a condition.
```

Comment:
document that yield get Record class see http://www.rubydoc.info/gems/yard/file/docs/Tags.md#yield